### PR TITLE
Add registry to each image from docker-compose

### PR DESCRIPTION
**Pull request by Victoremepunto**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Review by akofink — APPROVED** [review]
Thanks!

File: docker-compose.yml [hunk 1]
```diff
@@ -2,13 +2,13 @@
 version: "3"
 services:
   zookeeper:
-    image: confluentinc/cp-zookeeper
+    image: https://quay.io/repository/cloudservices/cp-zookeeper
```
Comment (akofink) [inline docker-compose.yml]: Technically, there is no http(s) schema. It's `docker://`. Also, these image urls should not include `/repository/`. Did you test this locally?

**Comment by akofink** [comment]
I've pushed a fix for my comments and will merge once tests pass.

File: docker-compose.yml (hunk 1 above)
Comment (Victoremepunto) [reply docker-compose.yml]: ah ! you're right - sorry , thanks for fixing this :)

**Pull request merged**